src/main/java/org/assertj/core/api/AbstractDoubleArrayAssert.java

```diff
@@ -224,6 +226,33 @@ public SELF contains(double... values) {
     return myself;
   }
 
+  /**
+   * Verifies that the actual array contains the values of the given array, in any order.
+   * <p>
+   * Example:
+   * <pre><code class='java'> double[] values = new double[] { 1.0, 2.0, 3.0 };
+   *
+   * // assertion will pass
+   * assertThat(values).contains(1.0, 3.0, 2.0)
+   *
+   * // assertions will fail
+   * assertThat(values).contains(1.0, 4.0);
+   * assertThat(values).contains(1.1, 2.1);</code></pre>
+   *
+   * @param values the given values.
+   * @return {@code this} assertion object.
+   * @throws NullPointerException if the given argument is {@code null}.
+   * @throws IllegalArgumentException if the given argument is an empty array.
+   * @throws AssertionError if the actual array is {@code null}.
+   * @throws AssertionError if the actual array does not contain the given values.
+   * @since 3.19.0
+   */
+  public SELF contains(Double[] values) {
```

Comment:
Please also add 
```java
public SELF contains(Double[] values, Offset<Double> precision)
```

Reply:
https://github.com/assertj/assertj-core/blob/c0f48d3858e047ed0421a36c435e6f1b5503edcb/src/main/java/org/assertj/core/api/AbstractDoubleArrayAssert.java#L249

It's already there

Reply:
https://github.com/assertj/assertj-core/blob/c0f48d3858e047ed0421a36c435e6f1b5503edcb/src/main/java/org/assertj/core/api/AbstractDoubleArrayAssert.java#L249

It's already there

Reply:
That one is with primitive `double[]` array as first parameter, we would need the boxed `Double[]` array